proto/datastream/v1/datastream.proto

```diff
@@ -0,0 +1,64 @@
+syntax = "proto3";
+
+package datastream.v1;
+
+option go_package = "github.com/0xPolygonHermez/zkevm-node/state/datastream";
+
+message Batch {
+    uint64 number = 1;
+    bytes local_exit_root = 2;
+    bytes state_root = 3;
+    uint64 fork_id = 4;
+    uint64 chain_id = 5;
+}
+
+message L2Block {
+    uint64 number = 1;
+    uint64 batch_number = 2;
+    uint64 timestamp = 3;
+    uint32 delta_timestamp = 4;
+    uint64 min_timestamp = 5;
+    bytes l1_blockhash = 6;
+    uint32 l1_infotree_index = 7;
+    bytes hash = 8;
+    bytes state_root = 9;
+    bytes global_exit_root = 10;
+    bytes coinbase = 11;
+}
+
+message Transaction {
```

Comment:
I assume are ordered by its execution in the datastream.
Probably not needed to add `txIndex` at transaction level, but it may be useful on othe rmodules.
Just leave here the comment for discussion